bundles/framework/mydata/resources/scss/personaldata.scss

```diff
@@ -0,0 +1,91 @@
+/* Account tab
```

Comment:
We should get rid of this scss and move the styling that is still required to the components with styled-components.